src/coordinated_workers/coordinator.py

```diff
@@ -963,6 +991,17 @@ def _reconcile_mesh_policies(self) -> None:
             logger=logger,
         )
 
+    def _reconcile_peers_relation(self):
+        relations: List[ops.Relation] = self.model.relations.get(self._coordinator_peers_relation, [])
+
+        for relation in relations:
+            if not self._local_ip:
+                continue
+
+            relation.data[self._charm.unit]["local-ip"] = self._local_ip
```

Comment:
are we still using the local-ip name? or are we only using hostname?
If all we're doing is replacing 'local-ip' with 'hostname', perhaps it's less disruptive to bite our tongues, call it a misnomer, and keep using local-ip but set it to a different value. (adding a comment to explain what's going on)